### ISSUE TITLE
Upload/Download progress check

### ISSUE THREAD

**Issue by shahnazari** [jstaf/onedriver#260]
Hey Jeff. Good job with providing this valuable and life saving app. I wonder if it could be possible in a way to check the progress of an upload or download task. I currently do it naively by looking at the network BW usage in System Monitor! Your reply is appreciated beforehand.

**Comment by jstaf** [comment]
Yeah probably the best way would be the method mentioned here (have a little icon with progress): https://github.com/jstaf/onedriver/issues/112#issuecomment-939304029

In the meantime though, you can check the logs - it will say "uploading bytes X of Y" with the amount of progress (if file is larger than 4MB, it is uploaded in chunks) and then "upload complete!" when done.
